packages/models/src/tenant/protobufConverterFromV2.ts

```diff
@@ -165,7 +165,7 @@ export const fromTenantV2 = (input: TenantV2): Tenant => {
     externalId: externalId.data,
     features: input.features.map(fromTenantFeatureV2),
     mails: input.mails.map(fromTenantMailV2),
-    kind: input.kind ? fromTenantKindV2(input.kind) : undefined,
+    kind: input.kind != null ? fromTenantKindV2(input.kind) : undefined,
```

Comment:
This was needed because `TenantKindV2.PA = 0 ` so when it was evaluated here `0` is considered `faltsy` and the kind was set to `undefined`

Reply:
🥲